src/boot/boot.c

```diff
@@ -1769,6 +1778,38 @@ static void config_load_type1_entries(
         }
 }
 
+static void config_load_smbios_entries(
+                Config *config,
+                EFI_HANDLE *device,
+                EFI_FILE *root_dir,
+                const char16_t *loaded_image_path) {
+
+        assert(config);
+        assert(device);
+        assert(root_dir);
+
```

Comment:
This should check for confidential computing and skip if it's enabled, for the usual reasons